src/test/compile-fail/gated-non-ascii-idents.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-extern crate bäz; //~ ERROR non-ascii idents
+extern crate core as bäz; //~ ERROR non-ascii idents
```

Comment:
After this PR, we load `extern crate`s (in `BuildReducedGraphVisitor`) before checking for non-ascii idents (in the gated feature checking pass).